std/src/main/scala/algebra/std/list.scala

```diff
@@ -28,7 +35,7 @@ class ListMonoid[A] extends Monoid[List[A]] {
 
   override def combineN(x: List[A], n: Int): List[A] = {
     val buf = mutable.ListBuffer.empty[A]
-      (0 until n).foreach(_ => x.foreach(buf += _))
+    (0 until n).foreach(_ => x.foreach(buf += _))
```

Comment:
if we are geeking out with perf, isn't a while loop more explicitly better? I mean, then we don't have the closure?

Again. The sadness...